sat/cli/jobstat/main.py

```diff
@@ -25,8 +25,9 @@
 """
 import logging
 import textwrap
+import re
```

Comment:
nit: alphabetize imports
```suggestion
import logging
import re
import textwrap
```